src/modules/users/users.service.ts

```
@@ -574,6 +573,44 @@ export class UserService {
     try {
       const savedDoc = await this.saveDoc(createUserDocDto);
       await this.writeToFile(createUserDocDto, userFilePath, savedDoc);
+
+      // Register watcher if imported_from is e-wallet or QR Code
```

ESLint

[error] Replace `······` with `↹↹↹`

(prettier/prettier)